pkg/package.go

```diff
@@ -161,3 +176,62 @@ func (p Package) MarshalText() string {
 func (p Package) String() string {
 	return fmt.Sprintf("package '%s'", p.PID.String())
 }
+
+func (p Package) Create(opts PackageCreateOpts) error {
+	state, err := p.State()
+	if err != nil {
+		return err
+	}
+	opts.PID = state.PID
+	_, err = p.manager.Create(opts)
+	return err
+}
+
+func (p Package) CreateWithChanged(opts PackageCreateOpts) (bool, error) {
+	state, err := p.State()
+	if err != nil {
+		return false, err
+	}
+	if !state.Exists {
+		opts.PID = state.PID
+		_, err = p.manager.Create(opts)
+		return true, err
+	}
+	return false, nil
+}
+
+func (p Package) UpdateFiltersWithChanged(filters []PackageFilter) error {
```

Comment:
where is boolean => for changed value?

don't pretend that it is implemented.. :D

Reply:
I mean methods with `WithChanged` should always return `bool`